requirements.txt

```diff
@@ -1,9 +1,11 @@
 asyncio==3.4.3
+bbae-invest-api==0.1.2
 chaseinvest-api==0.2.4
 discord.py==2.4.0
 fennel-invest-api==1.1.0
 firstrade==0.0.30
 GitPython==3.1.43
+pillow==10.4.0
```

Comment:
I don't see this used anywhere, is it needed in this repo? If it's required inside of the `bbae-invest-api package`, then specifying it in your `setup.py` should be enough

Reply:
Pillow is needed for the captcha image function for bbae and dspac

Reply:
Yeah but as long as it's defined in their setup.py files it will be installed automatically and doesn't need to be defined here

Reply:
Sounds good, me and BIG NUT new to this pypi thing, we weren't aware.